Android.mk

```
@@ -71,6 +73,10 @@ LOCAL_MODULE := otbr-agent
 LOCAL_MODULE_TAGS := eng
 LOCAL_SHARED_LIBRARIES := libdbus
 
+ifeq ($(ANDROID_NDK),0)
```

```suggestion
ifneq ($(ANDROID_NDK),1)
```
In case of that `ANDROID_NDK` is not defined?